usl_pipeline/cloud_functions/requirements.txt

```diff
@@ -9,6 +9,7 @@ click==8.1.7
 click-plugins==1.1.1
 cligj==0.7.2
 cloudevents==1.10.1
+configparser==7.0.0
```

Comment:
configparser is in the stdlib. We don't need this requirements entry.
It looks like that package is to provide the newest version of configparser to older versions of python.